src/Components/Search/SearchBar.tsx

```diff
@@ -302,6 +303,12 @@ export class SearchBar extends Component<Props, State> {
       name: "term",
     }
 
+    if (searchState.state.selectedPreviewIndex != null) {
```

Comment:
This is what makes the screen reader 'work'. Basically, when you've selected a preview item with the keyboard, hitting 'Enter' will navigate you to that item, and _not_ the suggestion item.

So, by setting this prop corresponding to the 'id' of the currently active preview item- you basically get it working!